Source/engine/render/cel_render.cpp

```diff
@@ -637,10 +637,17 @@ void CelDrawTo(const CelOutputBuffer &out, int sx, int sy, const CelSprite &cel,
 
 void CelClippedDrawTo(const CelOutputBuffer &out, int sx, int sy, const CelSprite &cel, int frame)
 {
+	const int upperEdge = sy - 128;

```

Comment:
Yeah, I though about it but wasn't sure that it would be appropriate since it would make a Cel format related function aware of screen `Y` coordinates. Also I hate passing things by reference :)

Reply:
I guess if we also hand `GetCelFrameClipped` the height of the output then it's not to much of a stretch to have it handle this.

Reply:
> I don't think you'd need to pass anything by reference.

It was in reference to having all of it done by `CelGetFrameClipped`, using a helper of cause avoids that.